vic/drivers/shared_image/src/vic_init.c

```diff
@@ -1106,7 +1107,9 @@ vic_init(void)
         }
         // handle the bare soil portion of the tile
         vidx = veg_con_map[i].vidx[options.NVEGTYPES - 1];
-        Cv_sum[i] += veg_con[i][vidx].Cv;
+        if (vidx != NODATA_VEG) {
+            Cv_sum[i] += veg_con[i][vidx].Cv;
+        }
```

Comment:
Lines 1112-1114 should be deleted, since the previous block will be allowed to loop over all classes.